example.js

```diff
@@ -1,16 +1,19 @@
+// remark-usage-ignore-next
+import {inspect} from 'util'
+
 // Dependencies:
-var u = require('unist-builder')
-var toc = require('.')
+import {u} from 'unist-builder'
+import {toc} from './index.js'
 
-// Given a mdast tree:
-var tree = u('root', [
+// Now running:
+const tree = u('root', [
   u('heading', {depth: 1}, [u('text', 'Alpha')]),
   u('heading', {depth: 2}, [u('text', 'Bravo')]),
   u('heading', {depth: 3}, [u('text', 'Charlie')]),
   u('heading', {depth: 2}, [u('text', 'Delta')])
 ])
 
-var table = toc(tree)
+const table = toc(tree)
 
 // Yields:
-console.log('javascript', require('util').inspect(table, {depth: 3}))
+console.log('javascript', inspect(table, {depth: 3}))
```

Comment:
This is probably more specific to `remark-usage` being used but..

I've never really been a fan of using `require` inline like this. ESM forces static imports to be defined at the beginning of the module.

Sure, there are dynamic imports. However, now that code path needs to be asynchronous